### DISCUSSION TITLE
Why my plot with featureplot is so little ?????

### DISCUSSION THREAD

**Discussion by Lea2609** [satijalab/seurat#7647] [Q&A]
Hey, 
I don't know why but my plot with the function FeaturePlot is very little while my plot on PCA (Dimplot) is normal.. The last week, i don't have this problem.. 

![capture](https://github.com/satijalab/seurat/assets/126888511/03d17afe-2890-48e0-aa7e-6842525c89b3)

My code is: FeaturePlot(Seurat_mat_J0, reduction = "pca",features="nCount_RNA",max.cutoff='q80')

**Comment by anemartinezlarrinaga2898** [comment]
Im having the same issue

**Comment by ShenyueQ** [comment]
Hi ,

I had the same issue and figured out this could be due to how featureplot determines xlim and ylim in lines 221 and 222 of the function:

> xlims <- c(floor(x = min(data[, dims[1]])), ceiling(x = max(data[, dims[1]])))
> ylims <- c(floor(min(data[, dims[2]])), ceiling(x = max(data[, dims[2]])))

If you modify these two lines by removing the floor and ceiling parts, it may solve the problem:

```
xlims <- c(min(data[, dims[1]]), max(data[, dims[1]]))
ylims <- c(min(data[, dims[2]]), max(data[, dims[2]]))
```

Hope this helps.
